### ISSUE TITLE
pjsip not able to register in only IPV6 network

### ISSUE THREAD

**Issue by meghaParmar** [saravanandm/pjsip2.5-g729codec#6]
I updated pjsip library version to 2.5.5 to add support for IPV6 network.
 In config_site.h file added this line 
`#define PJ_HAS_IPV6 1`

But still it is not able to register with IPV6 network. Is there any other settings I need to add or somewhere I am missing? Any help will be  appreciate.

Thanks.

**Comment by ismaiI1** [comment]
This problem is caused by this library? i tried IPv6 before and i made call with g729 codec.

**Comment by meghaParmar** [comment]
Yes If network is supporting both IPv4 and IPv6 at that time It gives problem . But works fine with IPv4 only and IPv6
